src/test/components/ThreadActivityGraph.test.js

```diff
@@ -178,43 +164,4 @@ describe('SelectedThreadActivityGraph', function() {
       );
     });
   });
-
-  /**
-   * For completeness, test that the ThreadStackGraph is clickable, as it is using
-   * a different method than the ThreadActivityGraph to select a call node.
-   */
-  describe('ThreadStackGraph', function() {
```

Comment:
Would it be possible to still keep that stack graph tests? Maybe we can change the timeline type with  `changeTimelineType` action(since `TrackThread` looks for that state to change to the stack graph view) and then still will be able to test this?

Reply:
I believe this was duplicated coverage from the `src/test/components/TrackThread.test.js` file. It was testing it specifically for the loupe view.